### DISCUSSION TITLE
I tried and failed to install this on a Pi 4B running Bullseye with an Arducam IMX519

### DISCUSSION THREAD

**Discussion by JackOnGH** [photobooth-app/photobooth-app#297] [Q&A]
I ran into a wall on one of the last steps with a **“Failed to build opencv-python”**

Any ideas what went wrong? 
I am thinking of trying again, ordering fresh hardware if I can be sure I'll be able to get it up and running.

Also, is there a set-up that's "most likely to succeed"? 
Like, should I get a Pi 5 instead of a 4B? 
And is one camera best over others (I had gotten the ArduCam because it seemed like it had the best resolution and lens for a photobooth)?

**Comment by mgineer85** [comment]
Please try with using this option. That works usually https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-prefer-binary
